README.md

```diff
@@ -13,83 +13,122 @@ Create baseline images and test for CSS regression during standard Ember tests u
 
 `ember install ember-cli-visual-acceptance`
 ### Configuration
-* You can modify the save directory in `ember-cli-build.js` by including
+
+#### Location of saved images
+
+  You can specify the location where the saved images will be stored (do not store under **`tests`** directory).  Include the following in your `ember-cli-build.js`:
+
+
+```javascript
+visualAcceptanceOptions: {
+  imageDirectory: '<folder name>'
+}
+```
+
+#### Target browser and OS version
+
+You can specify the exact version of the browser as well as the OS the test will be run against.  To determine the current version of your browser and OS, follow these steps (from [detect.js](https://github.com/benbscholz/detect/blob/master/src/detect.js#L6-L11)):
```

Comment:
We can remove the detectjs link. I think the following info you added remves the need for the link